src/py_obs/osc.py

```diff
@@ -15,8 +17,10 @@ def __str__(self) -> str:
 class Osc:
     username: str
     password: str
+    private_key_path: str
 
     api_url: str = "https://api.opensuse.org/"
+    ibs_url_about: str = "https://api.suse.de/about"
```

Comment:
This shouldn't be hardcoded, you can infer it from `self.api_url`
```suggestion
```

Reply:
I can obviously derive the value but I don't see the logic. These are two different URL I don't see why IBS one should be derived from OBS.

Reply:
When you use MFA with IBS, you have to also talk to the IBS API. Hence it makes no sense to have the ibs url hardcoded as then `api_url` is `https://api.suse.de`